src/PlusImageProcessing/Testing/vtkPlusTransverseProcessEnhancerTest.cxx

```diff
@@ -16,15 +16,41 @@ See License.txt for details.
 #include <vtkSmartPointer.h>
 #include <vtksys/CommandLineArguments.hxx>
 
+
+#include "PlusTrackedFrame.h"
+#include "vtkPlusTrackedFrameList.h"
+#include "vtkImageCast.h"
+
+
 //----------------------------------------------------------------------------
 
-int main(int argc, char **argv)
+
+int main(int argc, char** argv)
 {
+  bool printHelp = false;
+  vtksys::CommandLineArguments args;
+
+  //Setup variables for command line
+  std::string inputFileName;
+  std::string configFileName;
+  std::string outputFileName;
+  std::string linesImageFileName;
+  std::string intermediateImageFileName;
+  std::string shadowImageFileName;
+  std::string processedLinesImageFileName;
+
   int verboseLevel = vtkPlusLogger::LOG_LEVEL_UNDEFINED;
 
-  vtksys::CommandLineArguments args;
+  //Get command line arguments
   args.Initialize(argc, argv);
-
+  args.AddArgument("--help", vtksys::CommandLineArguments::NO_ARGUMENT, &printHelp, "Print this help");
+  args.AddArgument("--input-seq-file", vtksys::CommandLineArguments::EQUAL_ARGUMENT, &inputFileName, "The filename for the input ultrasound sequence to process.");
+  args.AddArgument("--config-file", vtksys::CommandLineArguments::EQUAL_ARGUMENT, &configFileName, "The filename for input config file.");
+  args.AddArgument("--output-seq-file", vtksys::CommandLineArguments::EQUAL_ARGUMENT, &outputFileName, "The filename to write the processed sequence to.");
+  args.AddArgument("--lines-image-file", vtksys::CommandLineArguments::EQUAL_ARGUMENT, &linesImageFileName, "Optional output files for subsampled lines input image");
+  args.AddArgument("--intermediate-image-file", vtksys::CommandLineArguments::EQUAL_ARGUMENT, &intermediateImageFileName, "Optional output file for intermediate data");

```

Comment:
intermediate is not specific enough, find out something more descriptive (it can happen that you'll want to export more "intermediate" data in the future and then you would need to change the argument names)